sushichef.py

```diff
@@ -50,8 +48,42 @@ class NewzBeatChef(YouTubeSushiChef):
         'CHANNEL_THUMBNAIL': CHANNEL_THUMBNAIL,
         'CHANNEL_DESCRIPTION': CHANNEL_DESCRIPTION,
     }
+    DATA_DIR = os.path.abspath('chefdata')
+    DOWNLOADS_DIR = os.path.join(DATA_DIR, 'downloads')
+    ARCHIVE_DIR = os.path.join(DOWNLOADS_DIR, 'archive_{}'.format(CONTENT_ARCHIVE_VERSION))
+
+    def get_video_ids(self):
+        return get_video_ids(NEWZ_BEAT_CHANNEL_ID)
+
+    def get_channel_metadata(self):
+        return {
+            'defaults': {
+                'license': licenses.AllRightsLicense("NewzBeat"),
```

Comment:
The spec lists CC BY-NC as the license, so we should use that here. (Also double-check to confirm it.)

Reply:
Updated to use `licenses.CC_BY_NCLicense`